.gitignore

```diff
@@ -1 +1,3 @@
 dist
+
+.idea/
```

Comment:
2/5 that editor configs should be in the global `gitignore `

Reply:
Fixed by https://github.com/mattermost/mattermost-plugin-gitlab/pull/280/commits/aea509ab9f6b1b9483a24114a693f64c2306ca7c